yarn-project/boxes/private-token/src/contracts/Nargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "private_token_contract"
+authors = [""]
+compiler_version = "0.1"
+type = "contract"
+
+[dependencies]
+aztec = { path = "./noir-aztec" }
+value_note = { path = "./value-note" }
```

Comment:
@sirasistant i think i can remove the copied dependencies and just do github links?

Reply:
https://github.com/sirasistant/example-contract/blob/main/Nargo.toml resolved